src/test/java/com/synopsys/integration/jira/common/rest/oauth1a/AuthenticateOAuthTest.java

```diff
@@ -59,60 +54,91 @@ public void createAuthUrl() throws Exception {
         URL url = new URL(authorizationUrl);
         assertTrue(StringUtils.isNotBlank(url.toString()));
 
-        System.out.println(authorizationToken);
-        System.out.println(authorizationUrl);
+        System.out.println("Please copy this temporary token into the authCredentials: " + authorizationToken);
+        System.out.println("Please follow this link and copy the verification code into the authCredentials: " + authorizationUrl);
+    }
+
+    // Set the two printed values here
+    private void setTemporaryTokenAndVerificationCode() throws Exception {
+        String temporaryToken = "";
+        String verificationCode = "";
+
+        if (StringUtils.isBlank(temporaryToken) || StringUtils.isBlank(verificationCode)) {
+            throw new Exception("Must set temporaryToken and verificationCode");
+        }
+
+        currentOAuthCredentials.setTemporaryToken(temporaryToken);
+        currentOAuthCredentials.setVerificationCode(verificationCode);
     }
 
     @Test
     @Ignore("Be sure you have all variables available before running")
     public void createAuthenticationToken() throws Exception {
+        setTemporaryTokenAndVerificationCode();
         JiraOAuthServiceFactory jiraOAuthServiceFactory = new JiraOAuthServiceFactory();
-        String jiraUrl = getEnvVarAndAssumeTrue(JiraCloudServiceTest.ENV_BASE_URL);
-        JiraOAuthService jiraOAuthService = jiraOAuthServiceFactory.fromJiraServerUrl(jiraUrl);
-
-        String temporaryToken = getEnvVarAndAssumeTrue(TEMPORARY_TOKEN);
-        String verificationCode = getEnvVarAndAssumeTrue(VERIFICATION_CODE);
-        String consumerKey = getEnvVarAndAssumeTrue(CONSUMER_KEY);
-        String privateKey = getEnvVarAndAssumeTrue(PRIVATE_KEY);
-        JiraOAuthGetAccessToken jiraOAuthGetAccessToken = jiraOAuthService.getJiraOAuthGetAccessToken(temporaryToken, verificationCode, consumerKey, privateKey);
+        JiraOAuthService jiraOAuthService = jiraOAuthServiceFactory.fromJiraServerUrl(currentOAuthCredentials.getBaseUrl());
+
+        JiraOAuthGetAccessToken jiraOAuthGetAccessToken = jiraOAuthService.getJiraOAuthGetAccessToken(
+            currentOAuthCredentials.getTemporaryToken(),
+            currentOAuthCredentials.getVerificationCode(),
+            currentOAuthCredentials.getConsumerKey(),
+            currentOAuthCredentials.getPrivateKey()
+        );
         OAuthCredentialsResponse oAuthCredentialsResponse = jiraOAuthGetAccessToken.execute();
 
         assertNotNull(oAuthCredentialsResponse);
         String token = oAuthCredentialsResponse.token;
         assertTrue(StringUtils.isNotBlank(token));
-        System.out.println(token);
+        System.out.println("Set this access token in the appropriate oauth env var: " + token);
+    }
+
+    // Add the new access token here if you don't want to use the env var (Should normally add the token to env var)
+    private void setAccessTokenIfProvided() {
```

Comment:
Should this be moved to the top of the test class so other user's know to set these values if they want to run these tests?

Reply:
For both of these I left them in the order you would want to set them, sort of like a step by step procedure. How would you prefer this class be laid out? For reference https://github.com/blackducksoftware/int-jira-common/blob/dc06f594a076a71ad3ff7cfb84fd395838c9b7c6/src/test/java/com/synopsys/integration/jira/common/rest/oauth1a/AuthenticateOAuthTest.java